presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -330,4 +333,31 @@ public QueryManagerConfig setInitializationTimeout(Duration initializationTimeou
         this.initializationTimeout = initializationTimeout;
         return this;
     }
+
+    public boolean isQueryAbridgingEnabled()
```

Comment:
from web UI perspective, it make sense to abridge always on the listing and show full query on the query page (or allow user to interactive opt-in to show the full query)
when the endpoint is consumed by someone else than the web UI, no abridging should take effect

also, this is `.../execution/QueryManagerConfig`, but it belongs more to some web UI config

Reply:
> maintaining semantics of BasicQueryInfo on /v1/query/ endpoint, for any future use?

There might be some usages of this endpoint that we don't know about. You should check with @electrum

Reply:
@electrum could you please take a look and make suggestions? Thanks.

Reply:
@nezihyigitbasi @findepi Modified the `QueryResource::getAllQueryInfo` to have a parameter for abridged version. So the behavior of `/v1/query` endpoint still remains the same without this optional parameter.

Reply:
@findepi The `query.abridging.enabled` config is no longer there. This configuration is provided by the UI when `/v1/query` is called. But I have still kept the queryAbridgedMaxLength in QueryManagerConfig, since the abridged version is created in QueryManager itself. Does that sound good? Or is it better to (1) create a new `UIConfig` class (2) add queryAbridgedMaxLength to it - and use this config class when constructing SqlQueryManager?